### DISCUSSION TITLE
When can JoinSet become stable?

### DISCUSSION THREAD

**Discussion by Cloud33** [tokio-rs/tokio#4757] [Q&A] [answered]
`JoinSet` -> `join_one`  It's too helpful for me. I can't wait to use it, but it's very troublesome to use it at present, so I want to know when it will be stable?

Strangely, I can compile it, but vs code displays error:
tokio = { version = "1.19.2", features = ["full"] }
![image](https://user-images.githubusercontent.com/8787198/172512819-f3fee715-49c1-4987-a017-907625b17ddf.png)

**Comment by Darksonn** [comment, marked as answer]
Maybe in two or three releases.

Your VS code error is because VS code is not set up with `--cfg tokio_unstable`.

**Reply by Cloud33** [reply]
> Maybe in two or three releases.

Great. I hope I can use it as soon as possible

> Your VS code error is because VS code is not set up with . `--cfg tokio_unstable`

I am in a project `Cargo.toml` add
```
[build]
rustflags = ["--cfg", "tokio_unstable"]
```
But it didn't work. Was I wrong?

**Reply by Cloud33** [reply]
Well, I'm sure I'm wrong. I should be on this path `.cargo/config.toml` , thank you

**Reply by Darksonn** [reply]
Yeah, it doesn't go in `Cargo.toml`.
